homeassistant/components/esphome/light.py

```diff
@@ -391,3 +406,13 @@ def min_mireds(self) -> int:
     def max_mireds(self) -> int:
         """Return the warmest color_temp that this light supports."""
         return round(self._static_info.max_mireds)
+
+    @property
+    def min_color_temp_kelvin(self) -> int:
+        """Return the warmest color_temp that this light supports, in Kelvin."""
+        return mired_to_kelvin(self._static_info.max_mireds)
+
+    @property
+    def max_color_temp_kelvin(self) -> int:
+        """Return the coldest color_temp that this light supports, in Kelvin."""
+        return mired_to_kelvin(self._static_info.min_mireds)
```

Comment:
```suggestion
        return round(1000000 / self._static_info.min_mireds)
```

Reply:
Internal function _mired_to_kelvin using the same computation added in commit 879575d1385c06588fbe2d86f39f22a71506e791